subprojects/smoke-ide-test/src/smokeIdeTest/groovy/org/gradle/ide/sync/IsolatedProjectsJavaProjectSyncTest.groovy

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.ide.sync
+
+import org.gradle.ide.sync.fixtures.IsolatedProjectsIdeSyncFixture
+import org.hamcrest.core.StringContains
+
+class IsolatedProjectsJavaProjectSyncTest extends AbstractSyncSmokeIdeTest {
+
+    private IsolatedProjectsIdeSyncFixture fixture = new IsolatedProjectsIdeSyncFixture(testDirectory)
+    /**
+     * To run this test locally you should have Android Studio installed in /Applications/Android Studio.*.app folder,
+     * or you should set "studioHome" system property with the Android Studio installation path,
```

Comment:
💅 This information should be readily available in the error message, instead of just **Android Studio home must be provided either auto downloading must be enabled**